### DISCUSSION TITLE
Please tell me your favorite programming language!

### DISCUSSION THREAD

**Discussion by YPanY** [NAIST-SE/AttractingContributors#186] [General]
I want to know which programming langue is most loved among us.
Please add your favorite programming language or framework to the list in my readme.md.

https://github.com/Pan-NLP-NAIST/AttractingContributors-best_programming_langue/blob/main/README.md

**Comment by t-matsumoto4758** [comment]
Added!
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by tktkbohshi** [comment]
added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Comment by xincanfeng** [comment]
added.
can you help by telling me your favirate color?
https://github.com/FengXincan/AttractingContributors-color

**Comment by hernessteve** [comment]
I don't have one. When it comes to programming or software development i always trust the professionals to do their job. I started to study in this field and can do some basic stuff now but when it comes to my business i work closely with one [software development company](https://forbytes.com/) for some time. We have a good trustworthy relationships, very happy i found them.

**Comment by WonderBoy113** [comment]
I am in love with a Python! I know it's an easy language and you can't write anything particularly progressive in it, but I just love it. By the way, does anyone else write on it besides me? Haven't seen new programs written purely in Python for a long time!

**Reply by CodyBra** [reply]
> I am in love with a Python! I know it's an easy language and you can't write anything particularly progressive in it, but I just love it. By the way, does anyone else write on it besides me? Haven't seen new programs purely written in Python for a long time!
Python is just a miracle, I don't understand why it is so underestimated. Just because he is light, he is good! It can be studied thoroughly and not have problems with constant edits. The last thing I saw in Python is move outlook archive to new computer [https://www.outlooktransfer.com/how-to-transfer-outlook-archive-files-to-a-new-computer/](https://www.outlooktransfer.com/how-to-transfer-outlook-archive-files-to-a-new-computer/). I bought a subscription specifically for my office only because this particular program was the only one written in our favorite programming language)

**Comment by kaircx** [comment]
help me ...
https://github.com/kaircx/AttractingContributors-favorite-language/
